### PR TITLE
project: add .env files for secret configuration

### PR DESCRIPTION
**Pull request by otaviopace**
## Description

Adds .env files for secret configuration.

This closes https://github.com/pagarme/ghostbusters/issues/136

## Your checklist for this pull request

:rotating_light: Please review this items for a good pull request. :four_leaf_clover:

1. I've read the project's [Contributing Guidelines](../CONTRIBUTING.md)
1. My commits are well written and follow [pagarme/git-style-guide](https://github.com/pagarme/git-style-guide)
1. My changes are well covered by **tests** and **logs**
1. I've updated the project docs (if needed)
1. I fell safe about this implementation
1. I feel comfortable with the code I wrote, and I'm not ashamed to show it to my friends

In a good pull request, everything above is true :relaxed:

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,11 @@
+DB_HOST={{get "DB_HOST"}}
```
Comment (claytonsilva) [inline .env]: lindão hein!!!!

Comment (otaviopace) [reply .env]: hahahha sim

File: src/bin/worker.js [hunk 2]
```diff
@@ -8,6 +8,11 @@ const makeLogger = makeFromLogger('boleto/worker')
 const queueItemId = defaultCuidValue('qi_')()
 const service = BoletoService({ operationId: queueItemId })
 
+if (process.env.NODE_ENV === 'production' && process.env.DOTENV_PATH) {
+  // eslint-disable-next-line global-require
```
Comment (mccraveiro) [inline src/bin/worker.js]: Why did you disable eslint here?
IMO you should require dotenv on the beginning of the file:

```js
const dotenv = require('dotenv')
...
if (process.env.NODE_ENV === 'production' && process.env.DOTENV_PATH) {
  dotenv.config({ path: process.env.DOTENV_PATH })
}
```

Comment (otaviopace) [reply src/bin/worker.js]: Ajustado :)

**Pull request merged**